runtime/src/lib.rs

```diff
@@ -353,6 +355,8 @@ parameter_type_with_key! {
 		// Dusting is handled by duster pallet.
 		// However, to make sure that account is reaped/killed and storage updated, ED must be > 0
 		// On ED = 0 - accounts are never reaped.
+
+		// We might want to set this to the same thing as duster e.g. 1000 for everything as a base
```

Comment:
@enthusiastmartin

Reply:
Yes, we can set all ED to 1000.  

The only is that there is not dust handler - so the dust will be lost at the moment.

Reply:
Let's actually do 1_000_000 everywhere. It's still 6 decimals so it's like 4 cents if you are dusting bitcoin. Anything else mostly is peanuts. We should do it dynamic tho asap.

Reply:
Maybe for now add default existential deposit to assetRegistry + set default deposits for generated assets to primitives/MIN_POOL_LIQUIDITY